sensors/count_sensor.py

```diff
@@ -40,7 +40,23 @@ def poll(self):
         data = self.es.search(index=self.index, body=query_payload)
 
         hits = data.get('hits', None)
-        if hits.get('total', 0) > self.count_threshold:
+        if hits is None:
```

Comment:
Would be great to see some unit tests for these new conditions.

Reply:
@nmaludy I was unable to find any tests apart from the bash scripts in `/test`. I tried to use Docker Compose but I believe that is broken because there is no `st2` image on DockerHub. From the CI output, it doesn't look like that script is being run. Where and how should I add tests?

Reply:
@AbhyudayaSharma Unit tests go in the `tests/` folder. These use the standard python `unittest2` framework. Some examples can be seen in the vSphere pack here: https://github.com/StackStorm-Exchange/stackstorm-vsphere/blob/master/tests/test_action_affinity_rule_create.py